web/src/message_list_view.ts

```diff
@@ -1733,21 +1861,21 @@ export class MessageListView {
         return $row;
     }
 
-    clear_trailing_bookend() {
+    clear_trailing_bookend(): void {
         const $trailing_bookend = this.$list.find(".trailing_bookend");
         $trailing_bookend.remove();
     }
 
     render_trailing_bookend(
-        stream_name,
-        subscribed,
-        deactivated,
-        just_unsubscribed,
-        can_toggle_subscription,
-        is_spectator,
-        invite_only,
-        is_web_public,
-    ) {
+        stream_name: string,
+        subscribed: boolean,
+        deactivated: boolean,
+        just_unsubscribed: boolean,
+        can_toggle_subscription: boolean,
+        is_spectator: boolean,
+        invite_only: boolean | undefined,
+        is_web_public: boolean | undefined,
```

Comment:
These last two might be values that would be nicer populate as always-boolean in a future revision.

Reply:
`render_trailing_bookend` is called by `message_list.js`, so I'd rather wait to futz with types until after we've converted `message_list`.

Reply:
Actually, it's not that hard to just add `?? false`, so I'll do that.